pkg/operator/configobservation/auth/auth_metadata.go

```diff
@@ -45,8 +38,7 @@ func ObserveAuthMetadata(genericListers configobserver.Listers, recorder events.
 		return observedConfig, errs
```

Comment:
Is the intent here is to overwrite `existingConfig` with empty config?

Reply:
I see that same pattern is used at couple of other places too. I am suspecting all these are intentional then?

Reply:
that depends maybe on the config. A non-existing auth config means not to configure auth at all. Also note that this is preexisting. I didn't touch that logic.